src/cmd/oui/update.rs

```diff
@@ -104,7 +104,10 @@ impl Update {
         let password = get_wallet_password(false)?;
         let wallet = load_wallet(opts.files)?;
         let keypair = wallet.decrypt(password.as_bytes())?;
-        let client = new_client(api_url(wallet.public_key.network));
+
+        let base_url: String = api_url(wallet.public_key.network);
+        let client = new_client(base_url.clone());
```

Comment:
spacing in inconsistent. i personally prefer this way

Reply:
Ok just changed the spacing and ran the formatting tool.